web/src/components/common.js

```diff
@@ -110,14 +110,39 @@ function chooseApprovalBadgeStyle (app, idx = 0) {
   }
 }
 
+class MergeableStatusBadge extends React.Component {
+  render () {
+    const mergeable = this.props.mergeable.toLowerCase()
+    switch (mergeable) {
+      case 'mergeable':
+        return <Badge variant="success">{mergeable}</Badge>
+      case 'conflicting':
+        return <Badge variant="warning">{mergeable}</Badge>
+      case 'unknown':
+        return <Badge variant="secondary">{mergeable}</Badge>
+      default:
+        return null
+    }
+  }
+}
+
+MergeableStatusBadge.propTypes = {
+  mergeable: PropTypes.string.isRequired
+}
+
 class ChangeStatus extends React.Component {
   render () {
     if (this.props.data.state === 'OPEN' && this.props.data.draft) {
       return <Badge variant="dark">Draft</Badge>
     }
     switch (this.props.data.state) {
       case 'OPEN':
-        return <span><Badge variant="success">Open</Badge> <small>[{this.props.data.mergeable.toLowerCase()}]</small></span>
+        return (
+          <span>
+            <Badge variant="success">Open</Badge>{' '}
+            <MergeableStatusBadge mergeable={this.props.data.mergeable} />
```

Comment:
you removed the lowercase call. I don't remember why it was there...

Reply:
Yes the removal is intentional.

Reply:
ok